client/src/app/components/page/task-detail/task-detail.component.html

```diff
@@ -1,18 +1,24 @@
-<div class="card mt-4 cardContainer flex-row">
+<div class="card mt-4 container flex-row">
     <div class="colCard container">
```

Comment:
you have container inside container
this is not right
please follow the common flow
container -> row -> col

Reply:
Okay, changed